### DISCUSSION TITLE
index.html + main.py example from the docs

### DISCUSSION THREAD

**Discussion by numpde** [idom-team/idom#922] [Problem]
I wasn't able to run the example with index.html + main.py from [here](https://idom-docs.herokuapp.com/docs/guides/getting-started/running-idom.html#embed-in-an-existing-webpage). After fixing some things in index.html, I get client-side:
```
WebSocket connection to 'ws://127.0.0.1:5001/_idom/_api/stream' failed: 
D @ idom-client-react.js:9
A @ idom-client-react.js:9
Ie @ idom-client-react.js:9
(anonymous) @ (index):24
```

Btw., is there the option to have idom-client-react.js unminified?

(idom==1.0.0a3)

**Comment by Archmonger** [comment]
Did you make sure to match your idom version in the `https://esm.sh/idom-client-react` string?

**Reply by numpde** [reply]
I see, that wasn't clear. It works now with `https://esm.sh/idom-client-react@1.0.0a3`.

**Reply by Archmonger** [reply]
Yeah, it's on our todo list to develop a template tag (equivalent to Django-IDOM)

See below:
- https://github.com/idom-team/idom/issues/653
